gitapp/lib/user_details/user_avatar_widget.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:gitapp/constants/constants.dart';
+import 'package:gitapp/user_details/user_details_model.dart';
+
+Row userAvatarWidget(UserDetails userMock) {
```

Comment:
Class names should start from uppercase so "UserAvatarWidget" would be a better name